lib/plugins/bridge.js

```diff
@@ -46,9 +46,17 @@ function options (env) {
     , minutes: env.extendedSettings.bridge.minutes || 1440
   };
 
+  var interval = env.extendedSettings.bridge.interval || 60000 * 2.5; // Default: 2.5 minutes
+
+  if (interval < 1000 || interval > 300000) {
```

Comment:
I bet 1000 would be too low, too

Reply:
Will not kill dexcom share to poll every second...

Reply:
but will ask quite some computing resources on the nightscout instance and thus stress the cpu usage with Azure, Heroku or self hosted quite a lot. Want to look at smarter ways of implementing this.